crates/languages/src/python/runnables.scm

```diff
@@ -41,6 +41,20 @@
     (#set! tag python-pytest-method)
 )
 
+; decorated pytest functions
+(
+    (module
+        (decorated_definition
+            (decorator)+ @decorator
+            definition: (function_definition
+                name: (identifier) @run @_pytest_decorated_method_name
```

Comment:
I've just noticed (after the merge :facepalm:) that you've used a different capture name here; I'm pretty sure it won't spawn any tasks? (Even though it'll show up in a gutter)
I'd suggest just using: `@_pytest_method_name`